src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -245,3 +246,37 @@ def _init_seen_tables(self):
 
     def _table_already_upgraded(self, target) -> bool:
         return target in self._seen_tables
+
+
+class TablesAnnotator:
```

Comment:
can you move it to `mapping.TablesMapping`? see https://github.com/databrickslabs/ucx/commit/65bffc55bd26fc1958cdfd2804a87c0e6c1d98ab#diff-231c20000db14946278e9cf9acf5e7ff20f91a527156def658929d962a9e754c

Reply:
Sure